tests/monitoring/prometheus_utils.go

```diff
@@ -278,3 +282,74 @@ func KillPortForwardCommand(portForwardCmd *exec.Cmd) error {
 	_, err := portForwardCmd.Process.Wait()
 	return err
 }
+
+func checkAlert(virtClient kubecli.KubevirtClient, alertName string) error {
+	alerts, err := getAlerts(virtClient)
+	if err != nil {
+		return err
+	}
+	for _, alert := range alerts {
+		if string(alert.Labels["alertname"]) == alertName {
+			return nil
+		}
+	}
+	return fmt.Errorf("alert doesn't exist: %v", alertName)
+}
+
+func verifyAlertExistWithCustomTime(virtClient kubecli.KubevirtClient, alertName string, timeout time.Duration) {
+	Eventually(func() error {
+		return checkAlert(virtClient, alertName)
+	}, timeout, 10*time.Second).Should(BeNil())
+}
+
+func verifyAlertExist(virtClient kubecli.KubevirtClient, alertName string) {
+	verifyAlertExistWithCustomTime(virtClient, alertName, 120*time.Second)
+}
+
+func waitUntilAlertDoesNotExist(virtClient kubecli.KubevirtClient, alertName string) {
+	Eventually(func() error {
+		alerts, err := getAlerts(virtClient)
+		if err != nil {
+			return err
+		}
+		for _, alert := range alerts {
+			if alertName == string(alert.Labels["alertname"]) {
+				return fmt.Errorf("alert exist: %v", alertName)
+			}
+		}
+		return nil
+	}, 5*time.Minute, 1*time.Second).Should(BeNil())
+}
```

Comment:
Using variadic would save you several for loops and repeated commands.
Also, in general, in utility function it is better to adjust the offset to ease the debugging
```suggestion
func waitUntilAlertsDoesNotExist(virtClient kubecli.KubevirtClient, alertNames ...string) {
	EventuallyWithOffset(1, func() error {
		alerts, err := getAlerts(virtClient)
		if err != nil {
			return err
		}
		for _, alert := range alerts {
			for _, alertName := range alertNames {
				if alertName == string(alert.Labels["alertname"]) {
					return fmt.Errorf("alert exist: %v", alertName)
				}
			}
		}
		return nil
	}, 5*time.Minute, 1*time.Second).Should(BeNil())
}
```
The same can be done with `verifyAlertExist`